packages/analytics/src/integrations/Omnitracking/__tests__/__snapshots__/Omnitracking.test.ts.snap

```diff
@@ -40,7 +40,9 @@ Object {
 
 exports[`Omnitracking track events definitions \`Checkout Step Editing\` return should match the snapshot 1`] = `
 Object {
+  "checkoutOrderId": undefined,
```

Comment:
Is the `checkoutOrderId` being `undefined` ok?

Reply:
Fixed it! 👍🏼